src/Cooked/Pretty/MockChain.hs

```diff
@@ -74,6 +74,11 @@ instance PrettyCooked MockChainError where
       <+> "but instead got:"
       <+> (case got of Nothing -> "none"; Just sHash -> prettyHash opts sHash)
   prettyCookedOpt _ (MCEUnsupportedFeature feature) = "Unsupported feature:" <+> PP.pretty feature
+  prettyCookedOpt _ (MCEPastSlot current target) =
+    "Unable to move back in time; current slot:"
```

Comment:
I like how this error message is like "oh very sorry for now this is not possible" rather than "of course you can't wait negative slots, what did you expect"